core/src/main/java/me/mykindos/betterpvp/core/effects/types/negative/ConcussedEffect.java

```diff
@@ -0,0 +1,22 @@
+package me.mykindos.betterpvp.core.effects.types.negative;
+
+import me.mykindos.betterpvp.core.effects.VanillaEffectType;
+import org.bukkit.potion.PotionEffectType;
+
+public class ConcussedEffect extends VanillaEffectType {
+
+    @Override
+    public String getName() {
+        return "Concuss";
+    }
+
+    @Override
+    public boolean isNegative() {
+        return true;
+    }
+
+    @Override
+    public PotionEffectType getVanillaPotionType() {
+        return PotionEffectType.SLOW_DIGGING;
```

Comment:
I dont think this actually slows down the players attack speed, since all that is handled by our plugins anyway

Reply:
It's not really there to slow down the player's attack speed but rather for aesthetic purposes. `Concussion.java` handles the messing-with-player's-attacks side of things.